src/applications/facility-locator/components/LocationHours.jsx

```diff
@@ -1,27 +1,61 @@
 import { values, every, capitalize } from 'lodash';
 import React, { Component } from 'react';
+import Raven from 'raven-js';
+import moment from 'moment';
 
 /**
  * VA Facility Known Operational Hours
  */
 export default class LocationHours extends Component {
-  colonizeTime(time) {
-    const found = time.match(/(\d?\d)(\d\d)(\w\w)/);
-    return `${found[1]}:${found[2]}${found[3]}`;
+  convertHour(hour, day) {
+    const hours = hour.split('-');
+    const isValid = hours.every(time => moment(time, 'hmmA').isValid());
+
+    if (!isValid) {
+      Raven.captureMessage('API data malformed', {
+        extra: {
+          data: this.props.location,
+          errorData: {
+            hours: { [day]: hour },
```

Comment:
You could probably get away with just including the location data and making the error message say that it's related to hours. Anyone looking at the sentry error should be able to figure out the error just by looking at the data or by putting it in the mock data. This would let you remove the `day` parameter in this function.